src/main.ts

```diff
@@ -16,7 +16,15 @@ let pr: PullRequest | undefined = undefined;
 try {
   const octokit = getOctokit(getInput('github_token', { required: true }));
 
-  pr = await PullRequest.initialize(context.issue.number, octokit);
+  if (!context.issue || !context.issue.number) {
```

Comment:
I don't follow this condition? Why we create a new Pull Request if it does not exist

Reply:
We create an "empty"/"uninitialized" Pull Request and pass it to the `action` and `post` function.

Pull Request object now provides a new method, `isInitialized,`, which returns a boolean and is used to determine if PR data is available. I think that this is cleaner then having everywhere if else conditions. Now it is in one place (`PullRequest.isInitialized(): boolean`)